package.json

```diff
@@ -34,7 +34,14 @@
     "eslint": "eslint . --max-warnings 0",
     "prettier": "prettier --write .",
     "whitepaper": "mkdir -p tmp_latex && pdflatex -output-directory=tmp_latex -interaction=nonstopmode -halt-on-error maiar.tex > /dev/null 2>&1 && pdflatex -output-directory=tmp_latex -interaction=nonstopmode -halt-on-error maiar.tex > /dev/null 2>&1 && mv tmp_latex/maiar.pdf . && rm -rf tmp_latex",
-    "prepublishOnly": "pnpm build"
+    "prepublishOnly": "pnpm build",
+    "docker:build": "docker build -t maiar-starter -t maiar-starter:latest -f Dockerfile .",
+    "docker:run": "docker container run -d --name maiar-starter --env-file maiar-starter/.env -p 3000:3000 maiar-starter:latest",
```

Comment:
Since you have docker-compose.yaml, use `docker compose build`... Docker compose is where you define the image name, so avoid using `-t` as well... And you can add the version the `image:` property so that releases should release the docker image

Reply:
I see you have the command up... if you want to maintain it you can just have `docker compose run -d`